pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/network-step/network-step.component.ts

```diff
@@ -279,4 +294,7 @@ export class SharedNetworkStepComponent extends StepFormDirective implements OnI
         this.formGroup.get('httpProxyPassword').setValue('');
         this.formGroup.get('httpsProxyPassword').setValue('');
     }
+    onNoProxyKeyup(event) {
```

Comment:
was this for testing, or do you still need it?

Reply:
Oops! missed this one. I will remove it.

Reply:
removed